core/src/main/java/org/infinispan/container/entries/ReadCommittedEntry.java

```diff
@@ -321,4 +327,31 @@ public boolean undelete(boolean doUndelete) {
       return false;
    }
 
+   public static class Externalizer extends AbstractExternalizer<ReadCommittedEntry> {
```

Comment:
Ok, it's fine if you can serialize it without the `ReadCommittedEntry`.

My thinking is that the user doesn't know what's in the `EntryView`, he only has an interface, so he shouldn't assume that it's serializable or that an `EntryView` is the optimal way to get what he wants back to the originator.